### PR TITLE
Append images to gifs

### PR DESCRIPTION
**Pull request by Jason717717**
Refactor create_gif and append_gif. All should be working.

### CODE REVIEW

**Comment by BrendanParmer** [comment]
Having trouble with `create_gif` commands like
```
$create_gif "https://cdn.discordapp.com/attachments/1089617135376011365/1089631866941886705/img.png" "https://upload.wikimedia.org/wikipedia/commons/thumb/3/30/Piebald_Squirrel_Pinto_Bean.jpg/1024px-Piebald_Squirrel_Pinto_Bean.jpg" https://www.survivethenews.com/wp-content/uploads/2022/08/1659378236_411_Reuters-Runs-Cover-for-Dementia-Joe-and-%E2%80%98Fact-Checks-Funny-Meme.jpg
```
Is this the correct usage of the command?

**Comment by BrendanParmer** [comment]
Might want to have error handling to print the usage for `Command raised an exception: TypeError: GifAppend.command() got multiple values for argument 'image_paths'`

File: utils/transformation.py [hunk 1]
```diff
@@ -188,4 +216,53 @@ async def image_compression(self, img_path, rate):
             im.save(new_file_name, quality=qlt)
             new_file_size = get_file_units(os.stat(new_file_name).st_size)
         old_file_size = get_file_units(old_file_size)
-        return (old_file_size, new_file_size, new_file_name)
\ No newline at end of file
+        return (old_file_size, new_file_size, new_file_name)
+
+class GifCreate(Command):
```
Comment (BrendanParmer) [inline utils/transformation.py:221]: Should probably move these to a separate GIFs cog or something

**Review by BrendanParmer — CHANGES_REQUESTED** [review]

File: utils/transformation.py (hunk 1 above)
Comment (Jason717717) [reply utils/transformation.py]: GifAppend.command() exception is now fixed, and gif commands are now moved to a GIF cog.

**Review by BrendanParmer — APPROVED** [review]

**Pull request merged**